operator/templates/stateful-set.yaml

```diff
@@ -133,6 +135,63 @@ spec:
             - name: jvm-options
               mountPath: /etc/cassandra/jvm.options
               subPath: jvm.options
+        {{ if eq .Params.METRICS_ENABLED "true" }}
+        - name: metrics
+          imagePullPolicy: Always
+          image: viivek46/cassandra:3.11.4
```

Comment:
Is it possible to parameterize the image so that we don't hardcode it in multiple places?

Reply:
Yes! I could swear I had created the JIRA for that, but apparently not. Just created https://jira.mesosphere.com/browse/DCOS-60288 for that. All Docker images and their pull policies should be configurable through parameters.